xml/resources-2.8.rng

```diff
@@ -65,14 +65,14 @@
     </element>
   </define>
 
-
   <define name="element-bundle">
     <element name="bundle">
-      <attribute name="id"><data type="ID"/></attribute>
-      <optional>
-        <attribute name="description"><text/></attribute>
-      </optional>
       <interleave>
+        <attribute name="id"><data type="ID"/></attribute>
+        <optional>
+          <attribute name="description"><text/></attribute>
+        </optional>
+        <ref name="element-resource-extra"/>
```

Comment:
Hence `instance_attributes` along with likely desired `meta_attributes`?

+

> also allows id and desc attributes to be interleaved

Attributes are interleaved by definition :)
We should rather lean towards making this more apparent rather than less (which I tried to start with (cad6b20).